### PR TITLE
Readme dev instr

### PR DESCRIPTION
**Pull request by eugen8**

### CODE REVIEW

**Review by stbanks — APPROVED** [review]

**Review by lilazn71 — APPROVED** [review]
Sweeeeeeeeeeeet!!

**Review by maihuynh91 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -3,3 +3,13 @@ Books swap and connection network
 started by a group of Grand View students in Web Application Development course.
 
 
+## Development process to make a change:
+0) git pull to make sure you have the latest code
+1) createa a branch for your card, e.g. `git checkout -b readme-dev-instr`  
+2) do your coding
```
Comment (Hienhtho) [inline README.md]: what do you mean coding here?

**Review by Hienhtho — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (eugen8) [reply README.md]: actual code changes.

Comment (eugen8) [reply README.md]: let me explain that a bit better

**Review by Hienhtho — APPROVED** [review]

**Review by eugen8 — COMMENTED** [review]
alright, ready to merge

**Pull request merged**